### PR TITLE
Updated the registering, added the volunteer sign up to event

### PR DESCRIPTION
**Pull request by fastboyz**

### CODE REVIEW

File: preplan-backend/config/db.config.js [hunk 1]
```diff
@@ -8,27 +7,86 @@ const initDB = () => {
...
-                    console.log("error", err);
+                    console.log("Error creating role", err);
                 }
             });
         }
     });
+
+    Skill.estimatedDocumentCount((err, count) => {
+        if (!err && count === 0) {
+
```
Comment (Maytaro) [inline preplan-backend/config/db.config.js:34]: How are we going to deal with the "Any departments" ? 
Add them as skill in DB and then treat it before sending it to the generator? 
Or Any departments is already as a concat of all skills?

File: preplan-backend/controllers/eventController.js [hunk 2]
```diff
@@ -373,9 +372,149 @@ router.get('/position/:id', [authJwt.verifyToken, authJwt.isOrganizer], (req, re
         });
 })
 
-router.post('/create/timeSlot', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
+router.get('/edition/:id/registered', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
```
Comment (Maytaro) [inline preplan-backend/controllers/eventController.js:375]: would like to keep the term inscription please, as registering is creating an account on the website.

File: preplan-backend/controllers/eventController.js [hunk 3]
```diff
@@ -373,9 +372,149 @@ router.get('/position/:id', [authJwt.verifyToken, authJwt.isOrganizer], (req, re
...
+            const picked = (({ inscriptionDate, status, plannerId }) => ({ inscriptionDate, status, plannerId }))(vol);
+            picked['profile'] = sanitizeProfile(vol.profile);
+            picked['edition'] = sanitizeEdition(vol.edition);
+            picked['availabilities']
+            picked['id'] = vol['_id'];
+            res.status(200).json(picked);
+        })
+});
+
+router.post('/event/register', [authJwt.verifyToken, authJwt.isOrganizer], async (req, res) => {
```
Comment (Maytaro) [inline preplan-backend/controllers/eventController.js:420]: would like to keep the term inscription please, as registering is creating an account on the website

File: preplan-backend/config/db.config.js (hunk 1 above)
Comment (fastboyz) [reply preplan-backend/config/db.config.js:34]: Any should be a list of everything

File: preplan-backend/controllers/eventController.js (hunk 2 above)
Comment (fastboyz) [reply preplan-backend/controllers/eventController.js:375]: Alrighty

File: preplan-backend/controllers/eventController.js (hunk 3 above)
Comment (fastboyz) [reply preplan-backend/controllers/eventController.js:420]: Alrighty

File: preplan-backend/controllers/eventController.js (hunk 2 above)
Comment (Maytaro) [reply preplan-backend/controllers/eventController.js:375]: is this not a method to send any inscription without filters on the status?

Comment (fastboyz) [reply preplan-backend/controllers/eventController.js:375]: well this gets all the register volunteers to an event

**Pull request merged**